workbench/tests/renderer/downloadmodal.test.js

```diff
@@ -167,6 +166,9 @@ describe('Integration tests with main process', () => {
     const dialogData = {
       filePaths: ['foo/directory'],
     };
+
+    // TODO: do we really want to mock electron.ipcRenderer.invoke
+    // or preload's api.electron.ipcRenderer.invoke?
```

Comment:
Oh yes, thanks. This is tricky but I think it might be okay to remove this comment and leave as-is. We are mocking all parts of `electron` during all tests with `__mocks__/electron.js`. So when our application code calls `window.Workbench.electron.ipcRenderer.invoke` , that in turn calls `electron.ipcRenderer.invoke`. So if the latter is mocked, then that's slightly more "real" code being run during tests than if we mocked the former.